taf/updater/updater_pipeline.py

```diff
@@ -343,7 +344,7 @@ def clone_remote_and_run_tuf_updater(self):
                 self.operation = OperationType.UPDATE
             else:
                 self.operation = OperationType.CLONE
-
+                
```

Comment:
You can install a VSCode extension which removes the whitespaces if you're using VSCode